### ISSUE TITLE
helpers are grouped by directory, not file?

### ISSUE THREAD

**Issue by jcowgar** [esdoc/esdoc#280]
I have

```
src/helpers/astring.js
src/helpers/adate.js
src/helpers/amath.js
```

Each has quite a few methods, but in the index, all the methods (which are exported) appear under one global header "helpers". I have no idea what is from the astring, adate, amath, etc... Further, when clicking on them, a very large document is loaded with all of my exported methods listed.

Is there some way to make esdoc group these by file, and not directory?

**Comment by typhonrt** [comment]
You're in luck... I just published the 0.0.3 update for [esdoc-plugin-enhanced-navigation](https://www.npmjs.com/package/esdoc-plugin-enhanced-navigation) adding grouping by file. 

This plugin only replaces the left-hand navigation with a much improved version.  As things go it doesn't change the references, variables, functions pages as I'll be making another plugin to make enhanced versions of those pages.

**Comment by h13i32maru** [comment]
ESDoc supports codes that are written ES2015 class base. So ESDoc does not target codes that has many functions. Please you can use ESDoc with class base codes.

However I consider this issue. Please check https://github.com/esdoc/esdoc/issues/363
